api/http/management.go

```diff
@@ -88,6 +88,11 @@ func (h *ManagementHandler) CreateIntegration(c *gin.Context) {
 		)
 		return
 	}
+	// we introduced a scope of integration and all the previous
+	// integrations are in the `deviceauth` scope
+	if len(integration.Scope) < 1 {
+		integration.Scope = model.ScopeDeviceAuth
+	}
```

Comment:
in here I will enforce the professional plan (somehow), so only those tenants can create inventory webhooks.